doc/nrf/gs_installing.rst

```diff
@@ -405,29 +407,29 @@ It also includes additional host tools, such as custom QEMU and OpenOCD builds.
          .. code-block:: console
 
             cd %HOMEPATH%
-            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.1/zephyr-sdk-0.14.1_windows-x86_64.zip
+            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.15.1/zephyr-sdk-0.15.1_windows-x86_64.zip
```

Comment:
```suggestion
            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.15.1/zephyr-sdk-0.15.1_windows-x86_64.zip
```

I wonder if we should be so specific on the version number here

Reply:
this change has actually already been merged via different PR